tasks/collect-mrrc-params/collect-mrrc-params.yaml

```diff
@@ -33,25 +36,28 @@ spec:
         DATA_FILE="$WORK_DIR/$(params.dataJsonPath)"
         MRRC_ENV_FILE_PATH="$(dirname $(params.dataJsonPath))/mrrc.env"
 
-        environment="$(jq -re '.mrrc.environment' $DATA_FILE)"
-        release="$(jq -re '.mrrc.release' $DATA_FILE)"
+        environment="$(jq -re '.mrrc.environment' "$DATA_FILE")"
```

Comment:
Can you update https://github.com/konflux-ci/release-service-catalog/blob/development/schema/dataKeys.json to list the mrrc keys?

Reply:
Added in new commit.